projects/demo/src/modules/utils/tokens/examples/8/index.html

```diff
@@ -0,0 +1,23 @@
+<p i18n>
+    Using <strong>TUI_NUMBER_FORMAT</strong> injection token you can customize
+    numbers formatting.<br />
+    For example: 10 500,33<br />
+    Can be customized as: 10_500.33<br />
+    <br />
+    <strong>Defaults:</strong>
+    <br />
+    decimalSeparator = '.'<br />
```

Comment:
Default is comma. You can print it from the injected token to make it automatic :)

Reply:
```
import {default as example8Html} from '!!raw-loader!./examples/8/index.html';
```
This file loaded as raw, I don't think it's possible